### ISSUE TITLE
Confusion aboout the "many to one" training style

### ISSUE THREAD

**Issue by zzwei1** [RodrigoGantier/SPADE_E2VID#2]
Nice work!
However, I'm a bit confused about the "many to one " training style. Does it mean that, in such a video reconstruction network, you have N predictions that  SPADE_E2VID outpued , while you only calculate the loss between the Nth prediction (last prediction in total N predictions) and the Nth ground truth?

**Comment by HaoDot** [comment]
hey zzwei1!
when I first get attach to this paper, the training style confuses me a lot. However,  after looking through for a few times, the following pic attracts me ,which shows how the loss function works.
![image](https://user-images.githubusercontent.com/75289266/129666693-27cdafe2-e720-4f51-9c02-24f7319a599b.png)

As u have mentioned above, i take it right
> such a video reconstruction network, you have N reconstructions that SPADE_E2VID outputed , while you only calculate the loss between the Nth reconstruction (the last reconstruction in total N reconstructions) and the Nth ground truth
